CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+#How to contribute
+TODO
+ 
+For the details on the Records Management project see[here.](https://community.alfresco.com/docs/DOC-6387-project-overview-records-management)
+
+##Getting started
+What you need to get started:
+
+* A [Jira](https://issues.alfresco.com/jira/projects/ALF/issues/?filter=allopenissues) account,
+
+* A [GitHub](https://github.com/) account
+
+Useful but not necessary:
+
+* An [Alfresco community](https://community.alfresco.com/) account - this is a good place to ask questions and find answers. 
+##Making changes
+
+We use Jira to track issues. If you are committing a fix for a raised issue please include the ticket number in both the merge requests and the git commit messages.
+For example a fix for the ticket [ALF-21953](https://issues.alfresco.com/jira/browse/ALF-21953) will have the following git commit message: "ALF-21953: updated to use super pom v9"
+
+If you are adding in a new feature or bug fix please do so [here.](https://issues.alfresco.com/jira/projects/ALF/issues/?filter=allopenissues) By raising a ticket in this project you will be agreeing to the Alfresco Contribution Agreement which can be found at the bottom of the 'Create Issue' form or alternatively attached to[this](https://community.alfresco.com/docs/DOC-7070-alfresco-contribution-agreement)page.
+
+When you are ready to make a change you just need to fork the [records-management](https://github.com/Alfresco/records-management) repository and then make your changes into your copy of the code.
+
+We have a set of standards we follow when writing code. These can be found [here.](https://community.alfresco.com/docs/DOC-4658-coding-standards)
+
+When formatting your change please make sure not to change the format of any other code as this can make the changes difficult to spot and please make sure to use the correct line ending (we use LF)
+
+We ask that when adding/changing code you also add/change the appropriate unit tests and that these tests all run before creating the pull request (these will be run as part of the request it just saves time if you know they will pass before hand). 
+
+###Writing translatable code
+TODO
```

Comment:
Can we remove this section and let the coding standards page cover it?

Reply:
Done